clients/gtest/gemm_gtest.cpp

```diff
@@ -44,6 +44,8 @@ Representative sampling is sufficient, endless brute-force sampling is not neces
 const
 vector<vector<int>> matrix_size_range = {
                                         {-1, -1, -1, -1, 1, 1},
+                                        { 3, 33,  3,  3, 3, 3},
```

Comment:
OK, understood; you probably need to get rid of the comments above matrix_size_range.  "Advance users only: BrainStorm the parameters but do not make artificial one which invalidates the matrix."  That is what confused me.